docs/CloudPermissions.md

```diff
@@ -0,0 +1,78 @@
+# Assign Required Cloud Permissions For Deploying Scalar DL
+
+This guide explains how to assign required permissions for the users to create cloud resources for deploying Scalar DL on Azure.
+
+From the security perspective, it is better to give users limited permissions to protect the environment from unwanted activities.
+
+## Azure
+
+The following JSON is a custom role that allows users to manage Azure cloud resources from the Azure portal or from command line tools for deploying Scalar DL.
+Please note that it is sufficient but not necessary since, it can/should be further restricted.
+
+In Azure Portal, you can create the role in the Subscriptions section. Choose your subscription and select Access control (IAM) from the menu, then click `+Add` and select `Add custom role`. 
+Once the role is created, you can assign it to a user or a group from `Add role assignment` in the `+Add` menu.
+
+Please replace your subscription ID in the `assignableScopes` array. You can give `roleName` and `description` at your convenience.
```

Comment:
I think it's a wrong usage of `at your convenience`.

Reply:
Changed it to `as per your convenience`